src/main/java/com/flightstats/hub/dao/aws/S3ConfigStrategy.java

```diff
@@ -82,8 +84,25 @@ static String getChannelTypedName(ChannelConfig config, String postfix) {
     private static BucketLifecycleConfiguration.Rule createRule(ChannelConfig config, String id) {
         return new BucketLifecycleConfiguration.Rule()
                 .withFilter(new LifecycleFilter(new LifecyclePrefixPredicate(id + "/")))
-                .withId(id)
+                .withId(BUCKET_LIFECYCLE_RULE_PREFIX.concat(id))
```

Comment:
It might make sense to move this concatenation up to `getChannelTypedName` where we're already doing some string manipulation on the `id`. That might merit a rename of that method.

Reply:
I thought about this. There could many callers of this method createRule(). Adding prefix logic here  ensures the prefix is added every time a lifecycle rule is created.